intl/hn/resources.md

```diff
@@ -0,0 +1,44 @@
+# # 100DaysOfCode के प्राथमिक संसाधन
+
+[# 100DaysOfCode अधिकृत वेबसाइट](http://100daysofcode.com/)
+
+### लेख
+1. [# 100DaysOfCode](https://medium.freecodecamp.com/join-the-100daysofcode-556ddb4579e4) freeCodeCamp माध्यम से जुड़ें
```

Comment:
This link doesn't work anymore (I need to update in the English version as well) my bad. Could you please change it to: https://www.freecodecamp.org/news/join-the-100daysofcode-556ddb4579e4/

Reply:
Updated